doc/cookbook/source/examples/distance/chi_square.rst

```diff
@@ -0,0 +1,35 @@
+===================
+Chi Square Distance
+===================
+
+The Chi Square Distance for real valued features x and x' extends the concept of :math:`\chi^{2}` metric to negative values.
```

Comment:
since you are using x and x' downstream, pls put them into (inline) math mode here as well

Reply:
We need more clarity here :-) The distance extends the concept of the metric, what does this mean? I think distance and metric are commonly used interchangeably. Explain specifically what is meant by extending to negative values.

Reply:
Can we have some updates here pls? :)

Reply:
@iglesias @karlnapf  The chi square distance is defined as:

![screenshot from 2018-06-08 15-30-10](https://user-images.githubusercontent.com/14959028/41160731-decb2e6a-6b30-11e8-9a8b-32d0713ff36c.png)

This is for only positive values. However, the implementation in shogun is for both positive and negative values, since the denominator is (|x_i|+|y_i|) so it extends the chi square distance to the negative values too. Is this more clear?

Reply:
To some extent. What reference are you using for this classical definition?

From I have read after checking shortly out there, a use case of this Chi-2 distance is to calculate distance between histograms. In that use case the values are positive of course. I suppose that people found other use cases where values are not necessary positive. I suspect that this dichotomy you have observed (definitions with and without the absolute value) may come from there.

It is fine for me if you want to leave the text as it is. I would not play with saying the distance extends the concept of metric to negative values since because I find it rather vague and potentially misleading.